plugins/scaffolder/src/alpha/index.ts

```diff
@@ -30,4 +30,6 @@ export {
   type ScaffolderFormFieldsApi,
 } from '@backstage/plugin-scaffolder-react/alpha';
 
+export { type FormField } from '@internal/scaffolder';
```

Comment:
Hmm, not sure about exporting this type from the `@internal/` package for `OpaqueTypes`.

I wonder if we should move this type into the `scaffolder-react` package instead?

@Rugvip thoughts?